next-app/src/lib/progress/types.d.ts

```diff
@@ -0,0 +1 @@
+type Id = string | number
```

Comment:
Is an `Id` actually ever a number? I don't see that happening anywhere.

Reply:
you're right, it hasn't been used like that but I had originally just left it open to be whatever a consumer might want to pass in to serve as an id.

Reply:
of course, I'm open to keeping this a string until there's a need for `number`